x-pack/plugins/apm/server/lib/settings/apm_indices/save_apm_indices.ts

```diff
@@ -27,9 +27,8 @@ export async function saveApmIndices(
 
 // remove empty/undefined values
 function removeEmpty(apmIndices: Partial<ApmIndicesConfig>) {
-  return Object.fromEntries(
```

Comment:
the reference to `Object.fromEntries` produced a typescript error (maybe it's not fully supported?) so i just converted it to a reduce.